src/index.js

```diff
@@ -0,0 +1,279 @@
+import './style.css';
+
+// Create image for more options
+import Vert from '../icons/vert.png';
+
+// Create trashcan image
+import TrashCan from '../icons/trashcan.png';
+
+// Create ToDoTasks object
+import ToDoTasks from './modules/tasks.js';
+
+const tasks = new ToDoTasks();
+
+if (localStorage.getItem('data') !== null) {
+  // If there is data stored, set collection to that data
+  tasks.list = JSON.parse(localStorage.getItem('data'));
+}
+
+const list = document.querySelector('ul');
+
+// Updates the indexes of each element
+function updateDomIndexes() {
+  const parent = document.querySelector('ul');
+  const elements = parent.querySelectorAll('li');
+  for (let i = 0; i < elements.length; i += 1) {
+    elements[i].className = i + 1;
+  }
+}
+
+function removeItem(event) {
+  let index = event.target.parentNode.className;
+  index = parseInt(index, 10);
+  tasks.removeTask(index);
+
+  // Removes element from DOM
+  event.target.parentNode.remove();
+
+  // Updates DOM indexes
+  updateDomIndexes();
+
+  // Updates local storage without the removed element
+  localStorage.setItem('data', JSON.stringify(tasks.list));
+}
+
+// Highlights the li element and displays the trashcan icon
```

Comment:
### Exporting/importing classes and functions as modules
As mentioned above, putting classes and functions in separate files brings multiple benefits